src/selfplay/game.cc

```diff
@@ -139,11 +139,11 @@ void SelfPlayGame::Play(int white_threads, int black_threads, bool training,
     auto best_eval = search_->GetBestEval();
     if (training) {
       // Append training data. The GameResult is later overwritten.
-      auto best_q = best_eval.first;
+      auto best_wl = best_eval.first;
```

Comment:
Would it be useful for q_ratio to train on Q instead of WL? Probably can be landed without deciding now.

Reply:
I thought as we stored D anyway, it with WL the training site will have more information and options.